src/frontend/admin/src/components/templates/orders/view/sections/OrderViewCertificateSection.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { ReactNode } from "react";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Unstable_Grid2";
+import TextField from "@mui/material/TextField";
+import { FormattedMessage, useIntl } from "react-intl";
+import { orderViewMessages } from "@/components/templates/orders/view/translations";
+import { Order } from "@/services/api/models/Order";
+
+type Props = {
+  order?: Order;
+  getViewIcon: (url: string) => ReactNode;
+};
+
+export function OrderViewCertificateSection({ order }: Props) {
+  const intl = useIntl();
+
+  if (!order?.certificate) {
+    return null;
+  }
+
+  return (
+    <Grid container spacing={2}>
+      <Grid xs={12} mt={2}>
+        <Typography variant="h6">
+          <FormattedMessage
+            {...orderViewMessages.certificateDetailsSectionTitle}
+          />
+        </Typography>
+      </Grid>
+      <Grid xs={12} sm={6}>
+        <TextField
+          data-testid="order-view-certificate-name"
+          fullWidth={true}
+          disabled={true}
+          label={intl.formatMessage(
+            orderViewMessages.certificateDefinitionName,
+          )}
+          value={order.certificate?.definition_title}
+        />
```

Comment:
Like other related fields, we should add a CTA here to go to certificate definition